course-mw/course-actors-common/src/main/java/org/sunbird/common/Common.java

```diff
@@ -0,0 +1,37 @@
+package org.sunbird.common;
+
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.request.Request;
+
+import java.util.Map;
+import java.util.HashMap;
+import java.util.List;
+
+public class Common {
```

Comment:
As mentioned in the previous comments @ https://github.com/sunbird-cb/sunbird-course-service/pull/8
The Class name must be changed from Common to something more related for this feature.